bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/client/HttpHelper.java

```diff
@@ -106,11 +108,13 @@ public static String getAuthorizationHeader(OAuthClientService oAuthClientServic
                     String lastToken = lastAccessToken;
                     if (lastToken == null) {
                         LoggerFactory.getLogger(HttpHelper.class).debug("The used access token was created at {}",
-                                accessTokenResponse.getCreatedOn().format(DateTimeFormatter.ISO_LOCAL_DATE_TIME));
+                                LocalDateTime.ofInstant(accessTokenResponse.getCreatedOn(), ZoneId.systemDefault())
```

Comment:
I generally prefer using the timezone setup by the user in OH as setting rather than the OS timezone.

Reply:
This is the minimum change needed for the API change without changing any binding specific logic. Previously the usage of `LocalDateTime` also relied on system's timezone, not `TimeZoneProvider`. Also, this is only logging, so a better change would be simply dropping conversion to local datetime and use default `Instant` formatting. This would however also change logic, so should not be done in context of this 1:1 API compatiblity fix PR.

Reply:
You're right  it was previously also based on system timezone.